test/over_react/component/abstract_transition_test.dart

```diff
@@ -745,7 +745,7 @@ class TransitionerComponent extends AbstractTransitionComponent<TransitionerProp
   List<TransitionPhase> transitionPhasesSet = [];
 
   @override
-  void setState(dynamic newState, [callback()]) {
+  void setState(dynamic newState, [Function callback]) {
```

Comment:
I think this change can be reverted now since this typing change is no longer present in `react-dart`.